Integrations/MicrosoftTeams/MicrosoftTeams.yml

```diff
@@ -221,11 +221,71 @@ script:
       name: member
       required: true
       secret: false
-    deprecated: false
+    deprecated: true
     description: Adds a member (user) to a private channel.
     execution: false
     name: add-user-to-channel
-  dockerimage: demisto/teams:1.0.0.1701
+  - arguments:
+    - default: false
+      description: The user to call.
```

Comment:
if i understand correctly, this is the display name? will email address work? need to clarify this in the description